lib/pushmi_pullyu/aip_downloader_files_directories.rb

```diff
@@ -0,0 +1,91 @@
+require 'rdf'
+require 'rdf/n3'
+require 'fileutils'
+
+# Download all of the metadata/datastreams and associated data
+# related to an object
+
+module PushmiPullyu::AipDownloaderFilesDirectories
+  FIXITY_PATH = '/content/fcr:fixity'.freeze
+  CONTENT_DATASTREAM_METADATA_PATH = '/content/fcr:metadata'.freeze
+  CHARACTERIZATION_PATH = '/characterization'.freeze
+  VERSIONS_PATH = '/content/fcr:versions'.freeze
+  CONTENT_PATH = '/content'.freeze
+  THUMBNAIL_PATH = '/thumbnail'.freeze
+  FEDORA3FOXML_PATH = '/fedora3foxml'.freeze
+  FEDORA3FOXML_METADATA_PATH = '/fedora3foxml/fcr:metadata'.freeze
+
+  FILENAME_PREDICATE = 'info:fedora/fedora-system:def/model#downloadFilename'.freeze
+
+  def self.included(_klass)
```

Comment:
The class was getting super-long and the logic was getting obscured by a lot of setup work, so I decided to spin this off as a module (not for reuse, but to keep semantically similar code/vars/constants together)